konfera/views.py

```diff
@@ -55,3 +58,24 @@ def event_details_view(request, event_slug):
     return render(request=request,
                   template_name='konfera/event_details.html',
                   context=context, )
+
+
+def cfp_form_view(request):
+    speaker_form = SpeakerForm(request.POST or None, prefix='speaker')
+    talk_form = TalkForm(request.POST or None, prefix='talk')
```

Comment:
yes that is good idea, i just mentioned it here as this was highlighted during the review, so i do not forget.